main/src/library/List.flix

```diff
@@ -1394,4 +1394,18 @@ mod List {
         toArray(rc, l) !> Array.shuffle(rnd) |> Array.toList
     }
 
+    ///
+    /// Returns the frequency for each element in list `l`
+    ///
+    pub def frequency(l: List[t]): Map[t, Int32] with Order[t] =
+        def freq(ll, m) = match ll {
```

Comment:
Perhaps the implementation could be replaced by a foldLeft?

Reply:
Hum... As a newbie in Flix (and more generally speaking in Functional Programming) I don't really not what is the foldLeft, foldRight, Monoid etc... But I'm learning...